operations/mimir-tests/test-multi-zone-distributor.jsonnet

```diff
@@ -0,0 +1,30 @@
+local mimir = import 'mimir/mimir.libsonnet';
```

Comment:
Instead of a new file, can you configure distributor multi-zone in `test-multi-zone.jsonnet`, please?

Reply:
I have moved it in ztest-multi-zone.jsonnet, as suggested. But, beside setting `multi_zone_distributor_enabled` and `multi_zone_distributor_availability_zones`, I have also enabled `autoscaling_distributor_enabled` because it is an important part of multi-AZ distributors. WDYT @pracucci?